src/test/java/org/cactoos/http/HtTimedWireTest.java

```diff
@@ -58,17 +59,16 @@ public void worksFine() throws Exception {
         );
     }
 
-    // @checkstyle MagicNumberCheck (1 line)
     @Test(expected = TimeoutException.class, timeout = 1000)
     @SuppressWarnings("PMD.AvoidUsingHardCodedIP")
     public void failsAfterTimeout() throws Exception {
-        // @checkstyle MagicNumberCheck (2 lines)
         final long timeout = 100;
-        final int port = 80;
         new HtTimedWire(
             new HtWire(
-                "192.168.0.0",
-                port
+                // non-routable IP address artificially creates a timeout error
+                // @see: https://stackoverflow.com/a/904609/3456163
```

Comment:
@iakunin I'm not sure using a javadoc annotation is a proper use of them in a normal comment like here, I think it's best if you remove it

Reply:
@victornoel you're right: that's not appropriate. Removed.